arbitrum-docs/stylus/stylus-gentle-introduction.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'A gentle introduction to Stylus'
+sidebar_label: 'A gentle introduction to Stylus'
+description: ''
+author: amarrazza
+sme: amarrazza
+target_audience: todo
+sidebar_position: 1
+---
+
+# A gentle introduction: Stylus
+
+import PublicPreviewBannerPartial from './partials/_stylus-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This introduction is for developers who are interested in learning about Stylus, a new way to write smart contracts in popular programming languages, like Rust, alongside the EVM.
```

Comment:
```suggestion
This introduction is for developers who want to build Ethereum decentralized apps (dApps) using Rust. This capability is made possible by **Stylus**, a new way to write EVM-compatible smart contracts using your favorite programming languages.
```

A suggestion, feel free to discard, nits welcome!

Rationale -> we want to open with the _**keywords that devs use to search for solutions to their problem**_ first, _then_ our solution to that problem.

Reply:
I am wary about saying "Ethereum" dapps because it implies the contracts are on L1 Ethereum and only for app builders. To me, the quickstart is specifically targeted at "developers who want to build Ethereum decentralized apps (dApps) using Rust" where this is more educational. I think only including Rust for the audience could be limiting too, since Stylus is about other languages and contributing libraries for non-supported languages as well. I do support giving Rust the spotlight though! 

How about this?

"This introduction is for developers who want to build on Arbitrum using popular programming languages, like Rust. This capability is made possible by **Stylus**, a new way to write EVM-compatible smart contracts using your favorite programming languages.

Reply:
I pushed a commit with the compromise, but happy to continue riffing on this